### PR TITLE
Remove fomantic header module

### PR DESCRIPTION
**Pull request by silverwind**
Likely still a few useless classes left, but I think I at least don't have missed any.

### CODE REVIEW

**Comment by silverwind** [comment]
I removed the `normal header` concept in favor of a single tailwind class to set font weight.

File: web_src/css/modules/header.css [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  height: 3em;
+  float: none;
+  display: block;
+  line-height: var(--line-height-default);
+  padding: 0;
+  margin: 0 auto 0.5rem;
+  opacity: 1;
+}
+
+.ui.header:not(h1):not(h2):not(h3):not(h4):not(h5):not(h6) {
```
Comment (delvh) [inline web_src/css/modules/header.css]:
```suggestion
.ui.header:not(h1,h2,h3,h4,h5,h6) {
```
should be supported sufficiently already (https://caniuse.com/css-not-sel-list),
especially compared to https://caniuse.com/mdn-css_selectors_not

File: web_src/css/modules/header.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+.ui.top.attached.header {
+  border-radius: 0.28571429rem 0.28571429rem 0 0;
+}
+
+.ui.bottom.attached.header {
+  border-radius: 0 0 0.28571429rem 0.28571429rem;
+}
+
+.ui.attached.header:not(h1):not(h2):not(h3):not(h4):not(h5):not(h6) {
```
Comment (delvh) [inline web_src/css/modules/header.css]:
```suggestion
.ui.attached.header:not(h1,h2,h3,h4,h5,h6) {
```

File: web_src/css/modules/header.css (hunk 1 above)
Comment (silverwind) [reply web_src/css/modules/header.css]: Interesting, I wasn't aware of that usage.

**Review by delvh — APPROVED** [review]

File: web_src/css/modules/header.css [hunk 3]
```diff
@@ -0,0 +1,168 @@
+.ui.header {
```
Comment (silverwind) [inline web_src/css/modules/header.css:4]:
```suggestion
/* based on Fomantic UI header module, with just the parts extracted that we use. If you find any
   unused rules here after refactoring, please remove them. */

.ui.header {
```

**Review by denyskon — APPROVED** [review]

**Pull request merged**